mm2src/adex_cli/src/scenarios/init_mm2_cfg.rs

```diff
@@ -311,13 +311,16 @@ impl Mm2Cfg {
     }
 
     #[inline]
-    fn inquire_hd_account_id(&mut self) -> Result<()> {
-        self.hd_account_id = CustomType::<InquireOption<u64>>::new("What is hd_account_id:")
-                .with_help_message(r#"Optional. If this value is set, the AtomicDEX-API will work in only the HD derivation mode, coins will need to have a coin derivation path entry in the coins file for activation. The hd_account_id value effectively takes its place in the full derivation as follows: m/44'/COIN_ID'/<hd_account_id>'/CHAIN/ADDRESS_ID"#)
-                .with_placeholder(DEFAULT_OPTION_PLACEHOLDER)
+    fn inquire_enable_hd(&mut self) -> Result<()> {
+        self.enable_hd = CustomType::<InquireOption<bool>>::new("What is enable_hd:")
+                .with_parser(OPTION_BOOL_PARSER)
+                .with_formatter(DEFAULT_OPTION_BOOL_FORMATTER)
+                .with_default_value_formatter(DEFAULT_DEFAULT_OPTION_BOOL_FORMATTER)
+                .with_default(InquireOption::None)
+                .with_help_message(r#"Optional. If this value is set, the AtomicDEX-API will work in HD wallet mode only, coins will need to have a coin derivation path entry in the coins file for activation. path_to_address `/account'/change/address_index` will have to be set in coins activation to change the default HD wallet address that is used in swaps for a coin in the full derivation path as follows: m/purpose'/coin_type/account'/change/address_index"#)
```

Comment:
1. "the Komodo DeFi API will work"
2. Should the template `/account'/change/address_index` be prepended with capital or small `m` like 'M/account/change...'
2. if name `path_to_address` is not stated yet perhaps `bip32_path` would better express the meaning

Reply:
Done